public/js/validation/register.js

```diff
@@ -0,0 +1,79 @@
+const registerForm = document.querySelector("#register-form");
+const registerFullName = registerForm.querySelector("[name='full_name']");
+const registerName = registerForm.querySelector("[name='user_name']");
+const registerPassword = registerForm.querySelector("[name='password']");
+const registerConfirmPass = registerForm.querySelector("[name='confirm_password']");
+
+let authRegister = new Validator();
+
+
+registerFullName.addEventListener("keyup", function (e) {
+    let checkEmptyFullName =
+        authRegister.checkEmptyField("registerFullName", messages.EMPTY_NAME, registerFullName);
+
+    if (checkEmptyFullName) {
+        let checkFullNameLength = authRegister.checkTextLength("registerFullName", messages.GET_FIELD_LENGTH(5, 30), registerFullName, 5, 30);
+    }
+})
+
+registerName.addEventListener("keyup", function (e) {
+    let checkSpecial;
+    let checkEmptyUserName = authRegister.checkEmptyField("registerName", messages.EMPTY_USER_NAME, registerName);
+    if (checkEmptyUserName) {
+        checkSpecial = authRegister.checkSpecialCharacters("registerName", messages.CHECK_SPECIAL_CHARCTERS, registerName);
+    }
+    if (checkSpecial) {
+        authRegister.checkTextLength("registerName", messages.GET_FIELD_LENGTH(6, 30), registerName, 6, 30);
+    }
+});
+
+registerPassword.addEventListener("keyup", function (e) {
+    let checkSpecial;
+    let checkEmptyPassword = authRegister.checkEmptyField("registerPassword", messages.EMPTY_PASSWORD, registerPassword);
+    if (checkEmptyPassword) {
+        checkSpecial = authRegister.checkSpecialCharacters("registerPassword", messages.CHECK_SPECIAL_CHARCTERS, registerPassword);
+    }
+    if (checkSpecial) {
```

Comment:
I think you miss understand at some point here. Please view my Validator class in models folder to see that Validator.checkEmptyField return true if it ALREADY have text on input, otherwise it return false, so my checkEmptyPassword return TRUE if only there are text in input field

Reply:
ohh okeyyy. my bad :+1: